linux/amzn/2014.09/foxpass_setup.py

```diff
@@ -148,9 +148,15 @@ def augment_sshd_config():
 # give "wheel" and "foxpass-sudo" groups sudo permissions without password
 def fix_sudo():
     os.system("sed -i 's/^# %wheel\tALL=(ALL)\tNOPASSWD: ALL/%wheel\tALL=(ALL)\tNOPASSWD:ALL/' /etc/sudoers")
-    if not file_contains('/etc/sudoers', 'foxpass-sudo'):
-        with open('/etc/sudoers', "a") as w:
+    if not file_contains('/etc/sudoers', '^#includedir'):
+        with open('/etc/sudoers', 'a') as w:
+            w.write('\n#includedir /etc/sudoers.d')
+    if not os.path.isdir('/etc/sudoers.d'):
```

Comment:
Well if it was a file, and this returned false, the mkdir would fail anyway. i would switch all of them to exists()